apps/test/unit/applab/designModeTest.js

```diff
@@ -0,0 +1,49 @@
+import {expect} from '../../util/configuredChai';
+import designMode from '@cdo/apps/applab/designMode';
+
+describe("appendPx", () => {
+  it('returns a valid css positive integer', function () {
+    const cssVal = designMode.appendPx(100);
+    expect(cssVal).to.equal("100px");
+  });
+  it('returns 0 as a valid value', function () {
+    const cssVal = designMode.appendPx(0);
+    expect(cssVal).to.equal("0px");
+  });
+  it('returns the given stringified integer as a valid value', function () {
+    const cssVal = designMode.appendPx("100");
+    expect(cssVal).to.equal("100px");
+  });
+  it('returns a value with px as a valid value', function () {
+    const cssVal = designMode.appendPx("100px");
+    expect(cssVal).to.equal("100px");
+  });
+  it('returns an empty string', function () {
```

Comment:
It would be nice if each of these test names described the type of input it's checking, so if you saw failed tests in a list, you would see at a glance what had broken:

```
✔ returns an empty string for "one hundred"
✔ returns an empty string for {object : 100}
✔ returns an empty string for ["bark", 300, 400]
✕ returns an empty string for [200, 300, 400]
✔ returns an empty string for no argument
✕ returns an empty string for null
✔ returns an empty string for undefined
```